### PR TITLE
feat: fromOpportunityId additional error logging

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

See https://shapeshift-dao.sentry.io/issues/241445/?query=is:unresolved&referrer=issue-stream&statsPeriod=7d&stream_index=8

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

None

## Testing

None

### Engineering

None

### Operations

None

### CODE REVIEW

File: src/pages/ThorChainLP/utils.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as Sentry from '@sentry/react'
```
Comment (woodenfurniture) [inline src/pages/ThorChainLP/utils.ts:1]: Likewise on this boi, named imports pleez

**Review by woodenfurniture — APPROVED** [review]
Also LGTM, with named imports being desirable

File: src/pages/ThorChainLP/utils.ts (hunk 1 above)
Comment (gomesalexandre) [reply src/pages/ThorChainLP/utils.ts]: e6f70eb16bc3414ecc3c18df860b030e9d11e5ee

Comment (gomesalexandre) [reply src/pages/ThorChainLP/utils.ts]: And obviously tested at runtime and confirmed Sentry still happy

**Pull request merged**